src/hooks/useRedirectMaintenance.ts

```diff
@@ -12,8 +13,9 @@ const useRedirectMaintenance = (startHours: number, endHours: number, redirectPa
   useEffect(() => {
     const checkAndRedirect = () => {
       const now = new Date();
-      const nextTime = new Date(now);
-      const hours = now.getHours();
+      const kstDate = toZonedTime(now, 'Asia/Seoul');
+      const nextTime = new Date(kstDate);
```

Comment:
```suggestion
      const nextKstTime = new Date(kstDate);
```
사소한 부분인데 이렇게 선언하면 읽기 더 편해질 거 같네요~

Reply:
수정했습니다 👍